src/assets/styles/main.scss

```diff
@@ -0,0 +1,105 @@
+*, *:before, *:after {
+    box-sizing: border-box;
+}
+
+html, 
+body, 
+div, 
+span, 
+object, 
+iframe, 
+figure, 
+h1, 
+h2, 
+h3, 
+h4, 
+h5, 
+h6, 
+p, 
+blockquote, 
+pre, 
+a, 
+code, 
+em, 
+img, 
+small, 
+strike, 
+strong, 
+sub, 
+sup, 
+tt, 
+b, 
+u, 
+i, 
+ol, 
+ul, 
+li, 
+fieldset, 
+form, 
+label, 
+table, 
+caption, 
+tbody, 
+tfoot, 
+thead, 
+tr, 
+th, 
+td, 
+main, 
+canvas, 
+embed, 
+footer, 
+header, 
+nav, 
+section, 
+video {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    font: inherit;
+    vertical-align: baseline;
+    text-rendering: optimizeLegibility;
+    -webkit-font-smoothing: antialiased;
+    text-size-adjust: none;
+}
+
+footer, 
+header, 
+nav, 
+section, 
+main {
+    display: block;
+}
+
+body {
+    line-height: 1;
+}
```

Comment:
`line-height: 1` is useful for texts inside buttons or in other custom components. Browsers has different initial values of `line-height` and it's worth to normalise it but the `1` is not a great choice in terms of readability ; )

Reply:
Just want to clear this up - should I change the unit in which the property is declared, or simply remove this rule?